packages/grid/x-data-grid/src/locales/nbNO.ts

```diff
@@ -0,0 +1,144 @@
+import { GridLocaleText } from '@mui/x-data-grid';
```

Comment:
Don't we need this one too?

```suggestion
import { nbNO as nbNOCore } from '@mui/material/locale';
import { GridLocaleText } from '@mui/x-data-grid';
```